LoopKitUI/Extensions/UIColor+LoopKit.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  UIColor.swift
+//  LoopKitUI
+//
+//  Created by Anna Quinlan on 7/1/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+import UIKit
+
+extension UIColor {
+    @nonobjc static let COBTintColor: UIColor = {
```

Comment:
[nit] consider ensuring this variable starts with lowercase per convention (e.g. `cobTintColor`, or better yet, spell out the acronym "COB")

Reply:
This was copy-pasted from what was in Loop when I migrated the `CorrectionRangeOverride` screen over; happy to change it though